docs/griptape-cloud/knowledge-bases/create-knowledge-base.md

```diff
@@ -0,0 +1,7 @@
+# Knowledge Bases
+
+Use [Knowledge Bases](https://cloud.griptape.ai/knowledge-bases) to prep your data ingested from [Data Sources](../data-sources/create-data-source.md) for retrieval. You can specify multiple Data Sources per Knowledge Base in order to access data ingested from different sources all in one place.
```

Comment:
I think we should be up front that KBs are what your application will be talking to, rather than individual DSs.